net/Socket.hpp

```diff
@@ -719,6 +719,16 @@ class SocketPoll
         wakeup(_wakeup[1]);
     }
 
+void speedwakeup()
```

Comment:
one-line comment to document why this one should be used would be helpful.
Also - a check that the lock is held that we expect to be locked here.